lgtm.yml

```diff
@@ -12,20 +12,39 @@ extraction:
       - aspell
       - aspell-en
       - libhiredis-dev
-      - libnl-3-dev
-      - libnl-genl-3-dev
-      - libnl-route-3-dev
-      - libnl-nf-3-dev
       - libzmq3-dev
       - libzmq5
       - swig3.0
       - libpython2.7-dev
       - libgtest-dev
       - dh-exec
       - doxygen
+      - cdbs
+      - bison
+      - flex
       - graphviz
     after_prepare:
-      - git clone https://github.com/Azure/sonic-swss-common; pushd sonic-swss-common; ./autogen.sh; fakeroot dpkg-buildpackage -us -uc -b; popd
+      - git clone https://github.com/Azure/sonic-buildimage; pushd sonic-buildimage/src/libnl3
+      - git clone https://github.com/thom311/libnl libnl3-3.5.0; pushd libnl3-3.5.0; git checkout tags/libnl3_5_0
+      - git apply ../patch/0001-mpls-encap-accessors.patch
+      - git apply ../patch/0002-mpls-remove-nl_addr_valid.patch
+      - ln -s ../debian debian
+      - fakeroot dpkg-buildpackage -us -uc -b
+      - popd
+      - dpkg-deb -x libnl-3-200_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-3-dev_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-genl-3-200_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-genl-3-dev_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-route-3-200_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-route-3-dev_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-nf-3-200_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - dpkg-deb -x libnl-nf-3-dev_3.5.0-1_amd64.deb $LGTM_WORKSPACE
+      - sudo dpkg -i libnl-3-dev_3.5.0-1_amd64.deb
+      - sudo dpkg -i libnl-genl-3-dev_3.5.0-1_amd64.deb
+      - sudo dpkg -i libnl-route-3-dev_3.5.0-1_amd64.deb
+      - sudo dpkg -i libnl-nf-3-dev_3.5.0-1_amd64.deb
+      - popd
+      - git clone https://github.com/qbdwlr/sonic-swss-common; pushd sonic-swss-common; git checkout azp; ./autogen.sh; fakeroot dpkg-buildpackage -us -uc -b; popd
```

Comment:
Why using a personal repo?

Reply:
I guess @qbdwlr  was trying out if this change will fix the LGTM analysis. Looks like it didn't.